projects/file-parsing/types-converting/json-struct.go

```diff
@@ -0,0 +1,33 @@
+package filesConverting
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Score struct {
+	Name       string
+	High_Score int32
```

Comment:
Ordinarily in Go I'd expect this field to be named `HighScore` not `High_Score` - can you rename it?